pkg/ephemeral-disk/ephemeral-disk.go

```diff
@@ -34,6 +35,13 @@ const (
 	ephemeralDiskPVCBaseDir = "/var/run/kubevirt-private/vmi-disks"
 )
 
+type DiskInfo struct {
```

Comment:
What's the significance of having the `DiskInfo` definition (as well as the `GetImageInfo()` function) in the `ephemeral-disk` package vs in the `container-disk` package? I can see it's used in both packages, as well as in others. Should it be in some shared utility package?

Reply:
> What's the significance of having the DiskInfo definition (as well as the GetImageInfo() function) in the ephemeral-disk package vs in the container-disk package?

Mentioned that in the PR description: I moved the definitions to avoid cyclic dependency between `ephemeral-disk` and `container-disk`.

> I can see it's used in both packages, as well as in others. Should it be in some shared utility package?

Hm... I got impression that  `ephemeral-disk` was such a 'shared utility package'. But not sure. Another candidate was `ephemeral-disk-utils` package but after checking its content (it contains `OwnershipManager` and related stuff) I though that `ephemeral-disk` would be the right place. Do you have suggestions where to move that instead?

Reply:
> Mentioned that in the PR description

Right, missed that!

TBH I'm not too familiar with these packages, but I got the impression that `ephemeral-disk` package will have code that mostly pertains to, well, ephemeral disks, and similarly, for the `container-disk` package. Looking at `ephemeral-disk-utils`, it strikes me more of a `disk-utils`, and may be a better home for the `DiskInfo` stuff. I know it's imported from other packages too (not sure how many), so renaming it might turn too noisy for this PR, so I leave it for your judgement :-)

Reply:
I would leave it as it is now just not to change the scope of the PR.